src/libutil/git.cc

```diff
@@ -314,7 +318,11 @@ Mode dump(
         return Mode::Symlink;
     }
 
-    case SourceAccessor::tMisc:
+    case SourceAccessor::tChar:
+    case SourceAccessor::tBlock:
+    case SourceAccessor::tSocket:
+    case SourceAccessor::tFifo:
+    case SourceAccessor::tUnknown:
     default:
         throw Error("file '%1%' has an unsupported type", path);
```

Comment:
```suggestion
        throw Error("file '%1%' has an unsupported type", path);
```
This can use `typeString()` now to show the type of the file.

Reply:
Done